src/pages/components/button/usage.mdx

```diff
@@ -13,17 +13,47 @@ Buttons are used to initialize an action. Button labels express what action will
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Button order</AnchorLink>
-<AnchorLink>Button alignment</AnchorLink>
-<AnchorLink>Labels</AnchorLink>
-<AnchorLink>Icon usage</AnchorLink>
-<AnchorLink>Danger button usage</AnchorLink>
-<AnchorLink>Variations</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Behavior</AnchorLink>
+<AnchorLink>Modifiers</AnchorLink>
+<AnchorLink>Related</AnchorLink>
 <AnchorLink>References</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Overview
+
+Buttons are clickable elements that are used to trigger actions. They communicate calls to action to the user and allow users to interact with pages in a variety of ways. Button labels express what action will occur when the user interacts with it.
+
+#### When to use
+
+Use buttons to communicate actions users can take and allow them to interact with the page. Each page should have one primary button, and any remaining calls-to-action should be represented as lower emphasis buttons.
```

Comment:
```suggestion
Use buttons to communicate actions users can take and to allow users to interact with the page. Each page should have one primary button, and any remaining calls-to-action should be represented as lower emphasis buttons.
```

Reply:
the last "When to use" comment I fixed manually in VS Code because for some reason it wouldn't let me commit it here, but it's updated anyway.